frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseSyncToParentByRebaseAction.java

```diff
@@ -262,9 +263,21 @@ private GitRebaseParams getIdeaRebaseParamsOf(GitRepository repository, IGitReba
     String newBaseBranchFullName = gitRebaseParameters.getNewBaseBranch().getFullName();
     String forkPointCommitHash = gitRebaseParameters.getForkPointCommit().getHash();
 
-    val options = kotlin.collections.SetsKt.hashSetOf(GitRebaseOption.INTERACTIVE, GitRebaseOption.KEEP_EMPTY);
+    val options = kotlin.collections.SetsKt.hashSetOf(GitRebaseOption.INTERACTIVE);
+
+    isGitRebaseOptionEntryAvailable("--empty=drop", gitVersion).map(options::add);
```

Comment:
Maybe `foreach` (if available) instead of `map`, if available... should have the same effect, but conveys the intent better (you're running it for **side effects**, not for the returned value)